examples/react/UIShell/src/App.tsx

```diff
@@ -266,6 +272,43 @@ function App() {
                     { label: 'Region', title: 'us-east-1 (N Virgina)' },
                   ]}
                 />
+                <ContainedList label="Profile links">
+                  <ContainedListItem
+                    renderIcon={User}
+                    onClick={() =>
+                      (window.location.href = 'https://example.com')
+                    }>
+                    User profile
+                  </ContainedListItem>
+                  <ContainedListItem
+                    renderIcon={IbmCloudKeyProtect}
+                    onClick={() =>
+                      (window.location.href = 'https://example.com')
+                    }>
+                    Access keys
+                  </ContainedListItem>
+                  <ContainedListItem
+                    renderIcon={Group}
+                    onClick={() =>
+                      (window.location.href = 'https://example.com')
+                    }>
+                    User management
+                  </ContainedListItem>
+                  <ContainedListItem
+                    renderIcon={Monster}
```

Comment:
Could we change this out for something else closer to plan and billing? Maybe the `Money` icon or similar?